nengo/spa/compare.py

```diff
@@ -45,32 +45,23 @@ def __init__(self, dimensions, vocab=None, neurons_per_multiply=200,
 
             self.inputA = nengo.Node(size_in=dimensions, label='inputA')
             self.inputB = nengo.Node(size_in=dimensions, label='inputB')
-            self.output = nengo.Node(size_in=dimensions, label='output')
+            self.output = nengo.Node(size_in=1, label='output')
 
         self.inputs = dict(A=(self.inputA, vocab), B=(self.inputB, vocab))
         self.outputs = dict(default=(self.output, vocab))
```

Comment:
Hmm, this output shouldn't have a vocab associated with it.  I think the right thing to do would be ```self.outputs = dict(default=(self.output, None))``` but I'm not sure if that breaks anything else....